### DISCUSSION TITLE
is there any dpr embedddings off-the-shelf for us to download?

### DISCUSSION THREAD

**Discussion by LzyFischer** [castorini/pyserini#910] [Q&A] [answered]
It seems that pyserini only provides retrieval results for dpr. however, could I obtain the dpr embeddings for the 21 million passsages here? thx!

**Comment by lintool** [comment, marked as answer]
We provide pre-built indexes that include pre-encoded vectors for the corpus in the original DPR paper. See here: https://github.com/castorini/pyserini/blob/master/docs/experiments-dpr.md

**Reply by LzyFischer** [reply]
thx a lot!
